### PR TITLE
Modify compute rel diff

### PR DESCRIPTION
**Pull request by huppd**

### CODE REVIEW

File: engine/tolerance.py [hunk 1]
```diff
@@ -46,24 +45,20 @@ def tolerance(stats_file_name, tolerance_file_name, member_num, member_type):
...
     ]
-    dfs.append(
-        parse_probtest_csv(stats_file_name.format(member_id="ref"), index_col=[0, 1, 2])
+    df_ref = parse_probtest_csv(
+        stats_file_name.format(member_id="ref"), index_col=[0, 1, 2]
     )
 
     ndata = len(dfs)
-    if ndata < 2:
+    if ndata < 1:
```
[the remark this answers is not in the document]
Comment (AnnikaLau) [reply engine/tolerance.py:53]: Ah now I got it!

File: scripts/generate_icon_testdata.sh [hunk 2]
```diff
@@ -12,19 +12,10 @@ if [[ -z "$PROBTEST_DATA" ]]; then
...
-            --file-id NetCDF "*lnd*.nc"
-        )
-    fi
-    if [[ "$exp" == "mch_opr_r04b07" ]]; then
-        file_id=(
-            --file-id NetCDF "*atm_3d_ml*.nc"
-        )
-    fi
+    file_id=(
+        --file-id NetCDF "*atm_3d_ml*.nc"
```
Comment (AnnikaLau) [inline scripts/generate_icon_testdata.sh:18]: Would it make sense to still have a test case with more than one file ID?

File: README.md [hunk 3]
```diff
@@ -198,7 +198,7 @@ Adjust the `.env` file to reflect the directory structure:
 
 `PROBTEST_REF_DATA`: path to the `probtest_data` subdirectory of the reference data.
 `PROBTEST_CUR_DATA`: path to the output directory you chose in the last step.
-`PROBTEST_TEST_EXPERIMENT`: full sets of reference data are generated for both the `mch_opr_r04b07` and `atm_amip_test` experiments. Chose one of the two or run both sequentially.
+`PROBTEST_TEST_EXPERIMENT`: full sets of reference data are generated for both the `mch_opr_r04b07` experiment. Chose one of the two or run both sequentially.
```
Comment (AnnikaLau) [inline README.md:201]:
```suggestion
`PROBTEST_TEST_EXPERIMENT`: full sets of reference data are generated for the `mch_opr_r04b07` experiment. 
```

Comment (AnnikaLau) [reply README.md:201]: Unless there will be a second test case...

File: scripts/generate_icon_testdata.sh (hunk 2 above)
Comment (huppd) [reply scripts/generate_icon_testdata.sh:18]: In principal yes. But I would like to make the testing independent of daint and of ICON, so I would prefer to do first that and then increase the test coverage. What do you think?

Comment (AnnikaLau) [reply scripts/generate_icon_testdata.sh:18]: Yes, we can do it like this.

Comment (AnnikaLau) [reply scripts/generate_icon_testdata.sh:18]:
```suggestion
    file_id=(
        --file-id NetCDF "*atm_3d_ml*.nc"
    )
```

Comment (AnnikaLau) [reply scripts/generate_icon_testdata.sh:18]: Still this needs to be fixed ;-)

**Review by AnnikaLau — APPROVED** [review]
LGTM now

**Pull request merged**